frontend/admin/src/js/components/user/ViewUser.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import {
+  HomeIcon,
+  DownloadIcon,
+  UserCircleIcon,
+} from "@heroicons/react/outline";
+import Breadcrumbs from "@common/components/Breadcrumbs";
+import type { BreadcrumbsProps } from "@common/components/Breadcrumbs";
+import PageHeader from "@common/components/PageHeader";
+import { Button } from "@common/components";
+import { Tab, TabSet } from "@common/components/tabs";
+import { commonMessages } from "@common/messages";
+import { useAdminRoutes } from "../../adminRoutes";
+import { User, useUserQuery } from "../../api/generated";
+import DashboardContentContainer from "../DashboardContentContainer";
+
+interface ViewUserPageProps {
+  user: User;
+}
+
+export const ViewUserPage: React.FC<ViewUserPageProps> = ({ user }) => {
+  const intl = useIntl();
+  const adminPaths = useAdminRoutes();
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Candidate Details",
+    description: "Title for the page when viewing an individual user.",
+  });
+
+  const userName = `${user?.firstName} ${user?.lastName}`;
+
+  const links = [
+    {
+      title: intl.formatMessage({
+        defaultMessage: "Home",
+        description: "Breadcrumb title for the home link.",
+      }),
+      href: adminPaths.home(),
+      icon: <HomeIcon style={{ width: "1rem", marginRight: "5px" }} />,
+    },
+    {
+      title: intl.formatMessage({
+        defaultMessage: "Users",
+        description: "Breadcrumb title for the all users table link.",
+      }),
+      href: adminPaths.userTable(),
+      icon: <UserCircleIcon style={{ width: "1rem", marginRight: "5px" }} />,
+    },
+    {
+      title: userName,
+    },
+  ] as BreadcrumbsProps["links"];
```

Comment:
Are these links supposed to match the breadcrumbs in the wireframe? Or am I looking at an outdated picture?
<img width="1332" alt="Screen Shot 2022-05-04 at 11 37 09 AM" src="https://user-images.githubusercontent.com/305339/166718206-aa28765a-a55d-4e14-9e6c-3f764d66f3a1.png">

Reply:
yeah, we were bouncing around on this, so far sticking with what we already have as a component for the time being I believe